package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "dotenv": "16.4.5",
     "dotenv-expand": "10.0.0",
+    "glob": "^11.0.0",
```

Comment:
You added a dependency to `package.json`, but did not commit the updated `package-lock.json`, so the tests are failing on CI. Please add the lock file.